ios/Podfile.lock

```diff
@@ -35,7 +35,7 @@ PODS:
   - Firebase/Performance (8.4.0):
     - Firebase/CoreOnly
     - FirebasePerformance (~> 8.4.0)
-  - FirebaseABTesting (8.6.0):
+  - FirebaseABTesting (8.10.0):
```

Comment:
Are all these updates required?

Reply:
I wasn't really sure. I'm going to say no though so I'll revert them 👍🏾